content/changelog-stable-old/index.html.haml

```diff
@@ -0,0 +1,28 @@
+---
+layout: changelog
+title: Archived LTS Changelog
+show_ratings: false
+has_rss: false
+actions:
+  <a class="app-button app-button--primary app-mobile-hide" href="/changelog-stable">
+    <ion-icon name="arrow-back-circle-outline"></ion-icon>
+    Back to Recent LTS Changelog
+  </a>
+---
+
+.ratings
+  - # Load older changelogs (all except the most recent 25, ordered correctly)
+  - archived_changelogs = site.changelogs[:lts].reverse.drop(25)
+  - archived_changelogs.each do |release|
+    %div
+      = partial('../_partials/release-header.html.haml', :release => release, :url => 'changelog-stable-old')
+      - if release.changes
+        %div.app-releases__list__items
+          = partial('../_partials/changelog-changes.html.haml', :changes => release.changes)
+      - else
+        %p No notable changes for this release.
+
+.app-banner
+  More recent LTS changelog of less than two years old can be found
```

Comment:
If the split between changelog pages is defined by a number of releases and not by time, it would be better to simplify this.

```suggestion
  This is the changelog archive. Changelog for recent releases can be found
```